test/utils/asserts.hpp

```diff
@@ -1,7 +1,23 @@
 #pragma once
 
+#include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+MATCHER(Trapped, "")
```

Comment:
Use it at least once somewhere.

Reply:
@gumb0 suggested `Traps` name and I agree it's better one.

Reply:
I think `Trapped` fits better, but don't care.